src/main/java/com/app/controllers/CompetitionController.java

```diff
@@ -0,0 +1,87 @@
+package com.app.controllers;
+
+import com.app.models.CompetitionModel;
+import com.app.models.ResultModel;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+public class CompetitionController {
+
+  ArrayList<CompetitionModel> competitions;
+
+  public void createNewCompetition() {
+    // TODO we need to get some clarification regarding how we read input
+  }
+
+  /**
+   * a method that returns a competition based on the ID provided
+   *
+   * @param id is the competition id for the competition you wish to return
+   * @return a competition based on the id that is provided
+   */
+  public CompetitionModel getCompetition(String id) {
+
+    for (CompetitionModel competition : competitions) {
```

Comment:
rename `competition` to `c`

Reply:
I dont think its that important. Move it to a low priority issue.